.github/README.md

```diff
@@ -61,7 +61,9 @@ Click on the "⭐ Star" button to help us gain more visibility on Github!
 
 ## Authors & Contributors
 
-This project exists thanks to the [authors](https://github.com/azerothcore/azerothcore-wotlk/blob/master/AUTHORS).
+The project was born in 2016 based on Sunwellcore. Unfortunately Sunwellcore was published without any git history, so on git there are no credits for all the contributors before 2016.
```

Comment:
Should it be SunwellCore instead of Sunwellcore? Or is this irrelevant

Reply:
@sudlud fixed, thanks